src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -202,7 +202,7 @@ interface RegExp {
 }
 
 interface RegExpConstructor {
-    [Symbol.species](): RegExpConstructor;
+    readonly [Symbol.species]: RegExpConstructor;

```

Comment:
[Per the spec](https://tc39.github.io/ecma262/#sec-get-regexp-@@species), the property type should be `this`.